shardingsphere-underlying/shardingsphere-executor/src/main/java/org/apache/shardingsphere/underlying/executor/sql/execute/jdbc/queryresult/MemoryQueryResult.java

```diff
@@ -81,7 +81,8 @@ private Object getRowValue(final ResultSet resultSet, final int columnIndex) thr
                 if (metaData.isSigned(columnIndex)) {
                     return resultSet.getLong(columnIndex);
                 }
-                return resultSet.getBigDecimal(columnIndex).toBigInteger();
+                Object result = resultSet.getBigDecimal(columnIndex);
```

Comment:
1. Object => BigDecimal
2. Generally we use 'result' to naming the return item in a function. Actually we don't return the 'result' here.

Reply:
Maybe result => bigDecimal

Reply:
thanks!